frontend/docs/css-style-guide.md

```diff
@@ -0,0 +1,301 @@
+# CSS Style guide
+
+## Purpose
+The aim of this document is to show good practices and conventions while using CSS in Wolox S.A.. This document will be used in trainings and as a reference in Code Reviewing.
+This document must be reviewed and regularly updated.
+
+## CSS Conventions
+
+### Naming:
+- When naming a class, use names as short as possible, but large enough to be representative:
+
+  For example: we consider that `.navigation` is too large, and we preffer to use `.nav` instead. On the other hand we think `.author` is more representative than naming your class `.atr`.
+
+- If a class name has more than one word, they should be separated by hyphens (hyphen-case a.k.a. kebab-case or spinal-case).
+
+  use `.demo-image` instead of `.demo_image`, `.demoImage` or `.demoimage`
```

Comment:
This should be right after the previous sentence, not in a paragraph of it's own